tests/integration/helm_fluentd_logs_test.go

```diff
@@ -0,0 +1,381 @@
+					},
+				),
+			),
+		).
+		Assess("fluentd metrics buffers PVCs are created",
+			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+				namespace := ctxopts.Namespace(ctx)
+				releaseName := ctxopts.HelmRelease(ctx)
+				kubectlOptions := ctxopts.KubectlOptions(ctx)
+
+				t.Logf("kubeconfig: %s", kubectlOptions.ConfigPath)
+				cl, err := k8s.GetKubernetesClientFromOptionsE(t, kubectlOptions)
+				require.NoError(t, err)
+
+				assert.Eventually(t, func() bool {
+					pvcs, err := cl.CoreV1().PersistentVolumeClaims(namespace).
+						List(ctx, v1.ListOptions{
+							LabelSelector: fmt.Sprintf("app=%s-sumologic-fluentd-metrics", releaseName),
+						})
+					if !assert.NoError(t, err) {
+						return false
+					}
+
+					return err == nil && len(pvcs.Items) == 3
+				}, waitDuration, tickDuration)
+				return ctx
+			}).
+		Assess("fluentd events statefulset is ready",
+			stepfuncs.WaitUntilStatefulSetIsReady(
+				waitDuration,
+				tickDuration,
+				stepfuncs.WithNameF(
+					stepfuncs.ReleaseFormatter("%s-sumologic-fluentd-events"),
+				),
+				stepfuncs.WithLabelsF(
+					stepfuncs.LabelFormatterKV{
+						K: "app",
+						V: stepfuncs.ReleaseFormatter("%s-sumologic-fluentd-events"),
+					},
+				),
+			),
+		).
+		Assess("fluentd events buffers PVCs are created",
+			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+				namespace := ctxopts.Namespace(ctx)
+				releaseName := ctxopts.HelmRelease(ctx)
+				kubectlOptions := ctxopts.KubectlOptions(ctx)
+
+				t.Logf("kubeconfig: %s", kubectlOptions.ConfigPath)
+				cl, err := k8s.GetKubernetesClientFromOptionsE(t, kubectlOptions)
+				require.NoError(t, err)
+
+				assert.Eventually(t, func() bool {
+					pvcs, err := cl.CoreV1().PersistentVolumeClaims(namespace).
+						List(ctx, v1.ListOptions{
+							LabelSelector: fmt.Sprintf("app=%s-sumologic-fluentd-events", releaseName),
+						})
+					if !assert.NoError(t, err) {
+						return false
+					}
+
+					return err == nil && len(pvcs.Items) == 1
+				}, waitDuration, tickDuration)
+				return ctx
+			}).
+		Assess("prometheus pods are available",
+			stepfuncs.WaitUntilPodsAvailable(
+				v1.ListOptions{
+					LabelSelector: "app.kubernetes.io/name=prometheus",
+				},
+				1,
+				waitDuration,
+				tickDuration,
+			),
+		).
+		Assess("fluent-bit daemonset is running",
+			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+				var daemonsets []appsv1.DaemonSet
+				require.Eventually(t, func() bool {
+					daemonsets = k8s.ListDaemonSets(t, ctxopts.KubectlOptions(ctx), v1.ListOptions{
+						LabelSelector: "app.kubernetes.io/name=fluent-bit",
+					})
+
+					return len(daemonsets) == 1
+				}, waitDuration, tickDuration)
+
+				require.EqualValues(t, 0, daemonsets[0].Status.NumberUnavailable)
+				return ctx
+			}).
+		Feature()
+
+	featMetrics := features.New("metrics").
```

Comment:
If this is a logs test (as name implies), then why do we test the metrics?

Reply:
that's a good point I'll remove the metrics tests and only keep the logs.